### PR TITLE
Jar file profile pic bug fix

### PR DESCRIPTION
**Pull request by jaivigneshvenugopal**

### CODE REVIEW

File: docs/UserGuide.adoc [hunk 1]
```diff
@@ -506,11 +506,54 @@ The `redo` command fails as there are no `undo` commands executed previously.
 `redo` (reapplies the `clear` command) +
 // end::undoredo[]
 
+// tag::setpath[]
+=== Setting the path to local profile pictures folder : `setpath`
+
+Sets the folder location to access and import debtors' profile pictures. +
+Paths should be absolute paths and not relative ones. Importantly, paths should point to the folder rather than the picture itself. +
```
Comment (lawwman) [inline docs/UserGuide.adoc]: Phrasing a little weird. "Importantly" feels out of place

**Review by lawwman — APPROVED** [review]
Seems fine to me.

File: docs/UserGuide.adoc [hunk 2]
```diff
@@ -506,11 +506,54 @@ The `redo` command fails as there are no `undo` commands executed previously.
...
+Examples:
+
+* `setpath C:/Users/acer/Desktop/SE/profilepic/` +
+* `setpath C:/Users/acer/Desktop/SE/profilepic` +
+* `setpath C:\Users\acer\Desktop\SE\profilepic\` +
+* `setpath C:\Users\acer\Desktop\SE\profilepic` +
+* `setpath out/production/resources/images/` +
+* `setpath out/production/resources/images` +
+* `setpath out\production\resources\images\` +
+* `setpath out\production\resources\images` +
```
Comment (khooroko) [inline docs/UserGuide.adoc]: have you confirmed that both / and \ will work?

**Review by khooroko — APPROVED** [review]
seems good

File: docs/UserGuide.adoc [hunk 3]
```diff
@@ -506,11 +506,54 @@ The `redo` command fails as there are no `undo` commands executed previously.
...
 
+// tag::setpath[]
+=== Setting the path to local profile pictures folder : `setpath`
+
+Sets the folder location to access and import debtors' profile pictures. +
+Paths should be absolute paths and not relative ones. Also, paths should point to the folder rather than the picture itself. +
+
+It is recommended to create a folder and fill it up with the necessary profile pictures for the application. If there is a need to add a new picture, it should be added into this specific folder. +
+
+The folder should only contain pictures with the `.jpg` extension. Moreover the pictures should be named after the debtors in the application's database. +
```
Comment (jelneo) [inline docs/UserGuide.adoc]: Moreover, *

File: preferences.json [hunk 4]
```diff
@@ -1,10 +1,10 @@
...
-    "windowWidth" : 1932.0,
-    "windowHeight" : 740.0,
+    "windowWidth" : 1294.6666259765625,
+    "windowHeight" : 694.6666870117188,
     "windowCoordinates" : {
-      "x" : 7,
-      "y" : 30
+      "x" : -6,
+      "y" : -6
     }
```
Comment (jelneo) [inline preferences.json]: This file shouldn't be pushed up right? Because the removed window width and height are set for 125% display

**Review by jelneo — COMMENTED** [review]
Please look through comments, especially the one about preferences

File: docs/UserGuide.adoc [hunk 5]
```diff
@@ -506,11 +506,54 @@ The `redo` command fails as there are no `undo` commands executed previously.
...
+
+For example, for the debtor Alex Yeoh, the name of his picture should be `AlexYeoh.jpg` +
```
Comment (khooroko) [inline docs/UserGuide.adoc]: her* since our ui pics show a lady's pic whenn Alex is selected

File: docs/UserGuide.adoc (hunk 3 above)
Comment (jaivigneshvenugopal) [reply docs/UserGuide.adoc]: noted.

File: preferences.json (hunk 4 above)
Comment (jaivigneshvenugopal) [reply preferences.json]: hmm i dont recall touching this, im not sure why it changed. Anyway i chnaged to the value that is present in our upstream repo.

Comment (khooroko) [reply preferences.json]: it was me i accidentally pushed it up once earlier tdy, just change it to whatever seems user friendly enough

File: docs/UserGuide.adoc (hunk 5 above)
Comment (jaivigneshvenugopal) [reply docs/UserGuide.adoc]: noted

**Review by khooroko — APPROVED** [review]

**Pull request merged**